installer/templates/phx_assets/package.json

```diff
@@ -1,26 +1,30 @@
 {
   "private": true,
+  "engines": {
+    "node": ">= 14.16.1"
+  },
   "scripts": {
-    "deploy": "webpack --mode production",
-    "watch": "webpack --mode development --watch"
+    "deploy": "webpack --mode=production",
+    "watch": "webpack --mode=development --watch"
```

Comment:
this also needs `--watch-options-stdin`?

Reply:
No. It's fine here without.
`mix phx.server` does not use this command directly (`npm run watch`).